code/parachain/frame/composable-maths/src/dex/constant_product.rs

```diff
@@ -466,6 +466,7 @@ pub fn compute_deposit_lp(
 /// https://github.com/ComposableFi/composable/blob/main/rfcs/0008-pablo-lbp-cpp-restructure.md#42-liquidity-provider-token-lpt-math-updates
 /// Equation 6
 pub fn compute_first_deposit_lp_<T: PerThing>(
+	// REVIEW(benluelo): Make this a named struct instead of a tuple?
```

Comment:
I thought we were getting rid of one of the parameters here?

Reply:
That PR is being held back by CI right now